commit 5b9da39dc58abcbdceaf9c2d283d0f64ece5bbdf
Author: Breno Leitao <leitao@debian.org>
Date:   Tue Jul 30 09:08:13 2024 -0700

    failcmd: add script file in MAINTAINERS
    
    failcmd is one of the main interfaces to fault injection framework, but,
    it is not listed under FAULT INJECTION SUPPORT entry in MAINTAINERS.  This
    is unfortunate, since git-send-email doesn't find emails to send the
    patches to, forcing the user to try to guess who maintains it.
    
    Akinobu Mita seems to be actively maintaining it, so, let's add the file
    under FAULT INJECTION SUPPORT section.
    
    Link: https://lkml.kernel.org/r/20240730160814.1979876-1-leitao@debian.org
    Signed-off-by: Breno Leitao <leitao@debian.org>
    Reviewed-by: Akinobu Mita <akinobu.mita@gmail.com>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index 8fab7a347fc0..fa9a13416f18 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -8541,6 +8541,7 @@ M:	Akinobu Mita <akinobu.mita@gmail.com>
 S:	Supported
 F:	Documentation/fault-injection/
 F:	lib/fault-inject.c
+F:	tools/testing/fault-injection/
 
 FBTFT Framebuffer drivers
 L:	dri-devel@lists.freedesktop.org